RecoTracker/LSTCore/src/alpaka/Event.h

```diff
@@ -183,7 +182,8 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     HitsBuffer<DevHost>& getHits(bool sync = true);
     HitsBuffer<DevHost>& getHitsInCMSSW(bool sync = true);
     ObjectRangesBuffer<DevHost>& getRanges(bool sync = true);
-    MiniDoubletsBuffer<DevHost>& getMiniDoublets(bool sync = true);
+    template <typename TSoA, typename TDev = Device>
+    typename TSoA::ConstView getMiniDoublets(bool sync = true);
```

Comment:
The template is probably not needed

Reply:
Unfortunately, it is needed. See the discussion in https://github.com/SegmentLinking/cmssw/pull/93#discussion_r1784757797